src/data/structure/Tuple2.java

```diff
@@ -0,0 +1,26 @@
+package data.structure;
+
+/**
+ * Supports simple pairs of arbitrary items
+ * @author benjaminposerow
+ *
+ * @param <T>
+ * @param <U>
+ */
+public class Tuple2<T,U> {
```

Comment:
Put them in a "util" package.  Not sure that is the best name. If "util" becomes too big I will split it.